core/tests/webdriverio_utils/ContributorDashboardPage.js

```diff
@@ -52,6 +56,18 @@ var ContributorDashboardPage = function() {
     '.e2e-test-question-suggestion-review-modal-header');
   var usernameContainer = $('.e2e-test-username');
   var reviewRightsDiv = $('.e2e-test-review-rights');
+  var selectorContainer = $('.e2e-test-language-selector');
+
+  var _openLanguageSelector = async function() {
+    await action.click('Test Language Selector Container', selectorContainer);
```

Comment:
Add a check to ensure the langaugeSelector is opened.

Reply:
Done. PTAL.